modules/ossm-config-control-plane-infrastructure-node.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-config-control-plane-infrastructure-node_{context}"]
+== Configuring all {SMProductShortName} Control Plane components to run on infrastructure nodes
+
+This task should only be performed if all of the components deployed by the {SMProductShortName} control plane (including Istiod, Ingress Gateway, and Egress Gateways) along with optional elements (such as Prometheus, Grafana, and Distributed Tracing) will run on infrastructure nodes. 
+
+If the control plane will run on a worker node, skip this task.
+
+.Procedure
+
+. Edit the `ServiceMeshControlPlane` resource to present the YAML file in the editor. In this example, `<name>` represents the name of the `ServiceMeshControlPlane` resource.
```

Comment:
Might be clearer to rephrase this step. Perhaps something like:
```
Open the `ServiceMeshControlPlane` resource as a YAML file in the editor:
```

I don't think the second sentence is really required - it's generally understood through the docs that variables are formatted this way. If you really want to include an explanation of the name field, I'd use callouts, e.g:

```
[source,terminal]
----
$ oc -n istio-system edit smcp <name> <1>
----
<1> `<name>` represents the name of the `ServiceMeshControlPlane` resource.
```
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output for more info re callouts